src/renderer/features/multisig-wallet-create/flexible-multisig/model/form-model.ts

```diff
@@ -116,8 +118,8 @@ const $existingProxy = combine(
           return a.connections.some(c => c.proxyAccountId === existingMultisig.accountId);
         }
 
-        if (accountUtils.isFlexibleProxiedAccount(a)) {
-          return a.proxyAccountId === existingMultisig.accountId;
+        if (accountUtils.isFlexibleMultisigAccount(a)) {
+          return a.multisigAccountId === existingMultisig.accountId;
```

Comment:
This looks suspicious - we were previosly cheching for `proxyAccountId`, not we are checking for `multisigAccountId`. Is this a bug fix?